db/migrate/20220212134157_add_area_fields_to_institution.rb

```diff
@@ -1,7 +1,13 @@
 class AddAreaFieldsToInstitution < ActiveRecord::Migration[7.0]
   def change
+<<<<<<< HEAD
```

Comment:
"Merge leftovers"

Reply:
I don't know why my ide didn't detect that while all tests passed locally